openedx/core/lib/request_utils.py

```diff
@@ -163,22 +161,22 @@ def process_exception(self, request, exception):
 # .. toggle_use_cases: opt_out
 # .. toggle_creation_date: 2021-03-11
 
-# .. toggle_name: EXPECTED_ERRORS[N]['LOG_ERROR']
+# .. toggle_name: IGNORED_ERRORS[N]['LOG_ERROR']
 # .. toggle_implementation: DjangoSetting
 # .. toggle_default: False
 # .. toggle_description: If True, the error will be logged with a message like: "Expected error ...".
 # .. toggle_use_cases: opt_in
 # .. toggle_creation_date: 2021-03-11
 
-# .. toggle_name: EXPECTED_ERRORS[N]['LOG_STACK_TRACE']
+# .. toggle_name: IGNORED_ERRORS[N]['LOG_STACK_TRACE']
 # .. toggle_implementation: DjangoSetting
 # .. toggle_default: False
 # .. toggle_description: If True, the stacktrace will be included with the logging message.
 # .. toggle_warning: Requires ``LOG_ERROR`` to be set to True, otherwise this value will be ignored.
 # .. toggle_use_cases: opt_in
 # .. toggle_creation_date: 2021-03-11
 
-# .. setting_name: EXPECTED_ERRORS[N]['REASON_EXPECTED']
+# .. setting_name: IGNORED_ERRORS[N]['IGNORED']
```

Comment:
1. This change:
```suggestion
# .. setting_name: IGNORED_ERRORS[N]['REASON_IGNORED']
```
2. Please search this PR for all cases of `reason_expected` or `REASON_EXPECTED` and change to `reason_ignored` or `REASON_IGNORED`.